src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -60,11 +72,15 @@ public boolean doesUserExist(final String userName) {
 
   @Override
   public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
+    updateUser(user, hashedPassword, hashedPassword.isBcrypted() ? "bcrypt_password" : "password");
```

Comment:
Because the `password` and `bcrypt_password` column names are used in so many places, it might be a good idea to extract them to class-level constants.